liblauncher/elevation.cpp

```diff
@@ -0,0 +1,22 @@
+#include <Windows.h>
+#include <jni.h>
+
+extern "C" JNIEXPORT jboolean JNICALL Java_net_runelite_launcher_FilePermissionManager_isRunningElevated(JNIEnv *env, jclass clazz, jlong pid) {
+	BOOL fRet = false;
+	HANDLE hToken = nullptr;
+	HANDLE handle = OpenProcess(PROCESS_ALL_ACCESS, TRUE, pid);
```

Comment:
`OpenProcess` can return a `nullptr` handle in case it runs into an error, you should check for that before continuing since `OpenProcessToken` expects a valid process handle